### PR TITLE
Updated operations specification documents (2021.1)

### PR DESCRIPTION
**Pull request by lazarevevgeny**
Remove incorrectly added files for Acosh-1, Asinh-1 and Atanh-1.
Updated TOC file

### CODE REVIEW

File: docs/doxygen/ie_docs.xml [hunk 1]
```diff
@@ -194,9 +192,13 @@
                         <tab type="user" title="PriorBoxClustered-1" url="@ref openvino_docs_ops_detection_PriorBoxClustered_1"/>
                         <tab type="user" title="PriorBox-1" url="@ref openvino_docs_ops_detection_PriorBox_1"/>
                         <tab type="user" title="Proposal-1" url="@ref openvino_docs_ops_detection_Proposal_1"/>
+                        <tab type="user" title="Proposal-4" url="@ref openvino_docs_ops_detection_Proposal_4"/>
                         <tab type="user" title="Range-1" url="@ref openvino_docs_ops_generation_Range_1"/>
+                        <tab type="user" title="Range-4" url="@ref openvino_docs_ops_generation_Range_4"/>
```
Comment (aalborov) [inline docs/doxygen/ie_docs.xml:197]: openvino_docs_ops_generation_Range_4 doesn't work

File: docs/doxygen/ie_docs.xml [hunk 2]
```diff
@@ -194,9 +192,13 @@
                         <tab type="user" title="PriorBoxClustered-1" url="@ref openvino_docs_ops_detection_PriorBoxClustered_1"/>
                         <tab type="user" title="PriorBox-1" url="@ref openvino_docs_ops_detection_PriorBox_1"/>
                         <tab type="user" title="Proposal-1" url="@ref openvino_docs_ops_detection_Proposal_1"/>
+                        <tab type="user" title="Proposal-4" url="@ref openvino_docs_ops_detection_Proposal_4"/>
```
Comment (aalborov) [inline docs/doxygen/ie_docs.xml:195]: doesn't work

**Review by aalborov — COMMENTED** [review]
2 links do not work - are those pages in the release branch yet? Link to the built website version http://openvino-docs.inn.intel.com/PR-2268/index.html
https://openvino-ci.intel.com/job/openvino/job/documentation/job/test/481/

File: docs/doxygen/ie_docs.xml (hunk 1 above)
Comment (lazarevevgeny) [reply docs/doxygen/ie_docs.xml:197]: Fixed. Thanks.

File: docs/doxygen/ie_docs.xml (hunk 2 above)
Comment (lazarevevgeny) [reply docs/doxygen/ie_docs.xml:195]: Fixed.

**Review by aalborov — APPROVED** [review]
Everything is ok now.

**Pull request merged**